AutomatedTesting/Gem/PythonTests/Physics/utils/physics_constants.py

```diff
@@ -0,0 +1,24 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+SPDX-License-Identifier: Apache-2.0 OR MIT
+Holds constants used across both hydra and non-hydra scripts.
+"""
+from sys import float_info
+
+"""
+Physx Event Bus
+"""
+GET_WORLD_TRANSLATION = "GetWorldTranslation"
```

Comment:
update: GetWorldTranslation is the `TransformBus`, might be worth moving to its own section